media/css/all.css

```diff
@@ -249,6 +249,11 @@ form input,form textarea {
     padding: 3px;
     font-size: 13px;
 }
+
+form#search_form  label {
```

Comment:
I'm looking into other places that we have a label inside a form. There's a rule earlier in this file which make this a `display: block`, however I've found a few places where this is overridden _again_ to make it inline. Looking at the list of template files using `<label>`:

* templates/follow/stream.html: overridden to inline-block here: https://github.com/MTG/freesound/blob/032cd6f03b1e500e28899d1d6b9a2e35a7d19d3d/media/css/all.css#L4106
* templates/api/minimal_registration.html: doesn't use all.css
* templates/accounts/recursive_file.html: doesn't use `form label` selector
* templates/accounts/registration.html: Uses display: block
* templates/search/advanced_search_form.html: your change, uses inline
* templates/forum/_section.html: Uses display: block

I wonder if it makes more sense to explicitly only use display: block on the pages where it's needed (only registration page, I think that forum advanced search can be inline or block-inline). As the default display for a label is inline anyway, we wouldn't need this explicit inline for the search form.

Does this make sense? Are you happy to try and make this change, or would you prefer that I do it?